plugins/user_mapping/src/local_file.hpp

```diff
@@ -0,0 +1,17 @@
+#ifndef IRODS_HTTP_API_LOCAL_FILE_HPP
+#define IRODS_HTTP_API_LOCAL_FILE_HPP
+
+#include <optional>
+
+#include <nlohmann/json.hpp>
+
+struct UserProfile {
+	std::string irods_user_name;
+	nlohmann::json attributes;
+};
+
+auto init(const nlohmann::json& _config) -> void;
+auto update() -> void;
```

Comment:
Must this be exposed? It appears to only be used in `local_file.cpp`.

Reply:
Removed header.